Sources/Task/TaskResult.swift

```diff
@@ -7,14 +7,23 @@
 //
 
 /// An enum for returning and propogating recoverable errors.
```

Comment:
Move this doc comment into the 3.0 block, neither types have documentation with this placement.